base/commandLine.cpp

```diff
@@ -1335,6 +1346,23 @@ static int recAddGames(const Common::FSNode &dir, const Common::String &engineId
 	return count;
 }
 
+static void calcMD5(Common::FSNode &path, long int length) {
+	if (!path.exists()) {
+		printf("Non-existent file path\n");
+	} else if (!path.isWritable()) {
+		printf("Non-writable file path\n");
```

Comment:
isWritable? we only need it for reading though

Reply:
I am so sorry. I did the change in #3745 but forgot to do the same here. Doing ASAP